.github/workflows/cd.yml

```
@@ -304,6 +304,8 @@ jobs:
         shell: bash
         working-directory: npm/git-cliff
         run: |
+          yarn config set npmAuthToken ${NODE_AUTH_TOKEN}
```

I'm thinking if we should use this from the secrets just to be safe. 

```suggestion
          yarn config set npmAuthToken ${{ secrets.NPM_TOKEN }}
```

What do you think?

---

Comes down to the same because that's where env gets it from too. This is how I always do it for my repos and I'm pretty sure it's also what you'll find in the GitHub workflow guides out there. I think it's whatever really. I can make the change if you really want but then I'll also drop the env binding a few lines below.

---

Ok sounds fair enough :)